www/config.rb

```diff
@@ -228,3 +228,4 @@ def render_markdown(text)
 redirect 'demo/build-system/index.html', to: '/demo/build-system/steps/1'
 redirect 'demo/process-supervisor/index.html', to: '/demo/process-supervisor/steps/1'
 redirect 'legal/index.html', to: '/legal/licensing'
+redirect 'cloudfoundry', to: '/get-started/cloudfoundry'
```

Comment:
I think this needs to be `redirect 'cloudfoundry/index.html', to: '/get-started/cloudfoundry' `. You can verify these locally by building and then separately serving the built site:

```
make build
cd build
php -S localhost:3000
```

and browse to http://localhost:3000/cloudfoundry